Lottery-Game-Chainlink-VRF.md

```diff
@@ -58,7 +58,7 @@ The official Chainlink Docs describe VRFs as:
 - After max number of players have entered the game, one winner is chosen at random
 - The winner will get `maxplayers*entryfee` amount of ether for winning the game
 
-## BUIDL IT
+## BUILD IT
```

Comment:
[Buidl](https://academy.binance.com/en/glossary/buidl) is an actual word in web3.

Reply:
reverted, thank you.